### PR TITLE
Converted G191E.docx to Markdown

### PR DESCRIPTION
**Pull request by ludomal**
Related to #40.

These markdown files will allow easier editing to the Recommendation. STL2009 text was used as base text.

The recommendation was split in three files (Main, Annex A, Annex B). These files will then be used by the Editor of G.191 to prepare the draft text to be submitted to the ITU.

The Markdown can be converted to HTML using pandoc. Then the generated HTML code can be open in MS Word. 

Pandoc can convert to docx directly however highlights seem to be lost in the process.

### CODE REVIEW

File: doc/g191.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+To clarify the use of the set of software tools, arranged as a software tool library, the ITU-T makes the following recommendations:
+
+1) The software tools specified in Annex A should be used as building modules of signal processing blocks to be used in the process of generation of ITU-T Recommendations, particularly those concerned with speech and audio coding algorithms.
+
+2) Some of the tools shall be used in procedures for the verification of interoperability of ITU‑T standards, mainly of speech and audio coding algorithms whose description is in terms of non-bitexact specifications.
+
+3) The use of these modules should be made strictly in accordance with the technical instructions of their attached documentation, and should respect the following terms.
+
+## 3. License and copyright<a name="license_and_copyright"></a>
```
Comment (dennisguse) [inline doc/g191.md]: Is the hyperlink necessary?

**Comment by dennisguse** [comment]
I really love to convert the docs to Markdown.
Some questions:
How do these related with the latex-based manual?
Do we need to put the Markdown files into a separate folder or would they also be fine under `manual`?

@ludomal I believe the pull request contains more commits than intended, or?

**Comment by ludomal** [comment]
Sorry... I created a new branch with a new pull request.
The latext-based manual is a different document from the recommendation. I think for clarity the sources should be kept in separate folders. Maybe we could move the manual folder in doc?

File: doc/g191.md (hunk 1 above)
Comment (ludomal) [reply doc/g191.md]: @dennisguse. Not really. I inserted them at the beginning as I had one big document. 
Now that it is split into three I can remove the hyperlinks if you think it is a bad idea.

**Pull request closed without merging**